### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Added all needed files for the documentation. Not sure if you already have some content to fill it. Will keep it a draft until you tell me how you want to proceed from here.

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Including .md file
+
+```console
+{@import <file path>}
+```
+
+Example:
+
+```console
+{@import ../../../../bindings/wasm/docs/api-reference.md}
```
Comment (Thoralf-M) [inline documentation/README.md]:
```suggestion
{@import ../README.md}
```

Comment (Dr-Electron) [reply documentation/README.md]: Why tho? It's just an example I want to remove at some point anyway :trollface: 
And please don't include READMEs in the docs 😅

Comment (Thoralf-M) [reply documentation/README.md]: It's just a not working example 😞

Comment (Dr-Electron) [reply documentation/README.md]: Haha I see. Then I will fix it just for you :trollface:

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**